linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -234,6 +239,13 @@ def install_kmp_user(inputfile, online=False):
 			elif ftype == KMFileTypes.KM_SOURCE:
 				#TODO for the moment just leave it for ibus-kmfl to ignore if it doesn't load
 				pass
+			elif ftype == KMFileTypes.KM_KMX:
+				# Sanitize keyboard filename if not lower case
+				kmx_id = re.sub('.kmx', '', f['name'])
+				for index, kb in enumerate(keyboards):
+					if kmx_id.lower() == kb['id'] and kmx_id != kb['id']:
+						os.rename(os.path.join(packageDir, f['name']), os.path.join(packageDir, kb['id']+'.kmx'))
```

Comment:
We need to confirm that it is ok leaving kmp.json to refer to the original filename after renaming - I don't think ibus-keyman uses that but needs checked

Reply:
@mcdurdin - should we only have Developer create/modify kmp.json files?

Reply:
I don't think we should be modifying kmp.json or the contents of a KMP file in Keyman for Linux or any of the clients, really. Issues such as these should be addressed in the source packages rather than bandaiding them in the client apps.

At best, if we do have to do this, we need to log it.